README.md

```diff
@@ -26,6 +26,28 @@ If you need support for other Google APIs, please check out the [Google APIs Cli
 $ composer require google/cloud
 ```
 
+## Caching Access Tokens
```

Comment:
This should not be at the top of the README. Can you move it to after the examples?